src/std/fs.ab

```diff
@@ -108,3 +108,34 @@ pub fun glob_multiple(paths: [Text]): [Text]? {
 pub fun glob(path: Text): [Text]? {
     return glob_multiple([path])?
 }
+
+/// Extract the file detecting from the filename the extension
+/// Supports: bz2, gz, xz, bz2, deb, rar, rpm, tar(gz/xz/bz), zip(war/jar), 7z
+/// Note: Not all the commands supports the output folder path
+pub fun extract(path: Text, target: Text): Bool{
+    if file_exist(path) {
+        if {
+            contains_any_regex(path, ["/.tar.bz2$/g", "/.tbz$/g", "/.tbz2$/g"]): trust $ tar xvjf "{path}" -C "{target}" $
+            contains_any_regex(path, ["/.tar.gz$/g", "/.tgz$/g"]): trust $ tar xzf "{path}" -C "{target}" $
+            contains_any_regex(path, ["/.tar.xz$/g", "/.txz$/g"]): trust $ tar xJf "{path}" -C "{target}" $
+            contains_any_regex(path, ["/.bz2$/g"]): trust $ bunzip2 "{path}" $
+            contains_any_regex(path, ["/.deb$/g"]): trust $ dpkg-deb -xv "{path}" "{target}" $
+            contains_any_regex(path, ["/.gz$/g"]): trust $ gunzip "{path}" $
+            contains_any_regex(path, ["/.rar$/g"]): trust $ unrar x "{path}" "{target}" $
+            contains_any_regex(path, ["/.rpm$/g"]): trust $ rpm2cpio "{path}" | cpio -idm $
+            contains_any_regex(path, ["/.tar$/g"]): trust $ tar xf "{path}" -C "{target}" $
+            contains_any_regex(path, ["/.xz$/g"]): trust $ xz --decompress "{path}" $
+            contains_any_regex(path, ["/.7z$/g"]): trust $ 7z -y "{path}" -o "{target}" $
+            contains_any_regex(path, ["/.zip$/g", "/.war$/g", "/.jar$/g"]): trust $ unzip "{path}" -d "{target}" $
```

Comment:
@Mte90 what if user doesn't have the command to extract the file archive? I think that we should fail and not trust the command that it will work:

```suggestion
            contains_any_regex(path, ["/.tar.bz2$/g", "/.tbz$/g", "/.tbz2$/g"]): $ tar xvjf "{path}" -C "{target}" $?
            contains_any_regex(path, ["/.tar.gz$/g", "/.tgz$/g"]): $ tar xzf "{path}" -C "{target}" $?
            contains_any_regex(path, ["/.tar.xz$/g", "/.txz$/g"]): $ tar xJf "{path}" -C "{target}" $?
            contains_any_regex(path, ["/.bz2$/g"]): $ bunzip2 "{path}" $?
            contains_any_regex(path, ["/.deb$/g"]): $ dpkg-deb -xv "{path}" "{target}" $?
            contains_any_regex(path, ["/.gz$/g"]): $ gunzip "{path}" $?
            contains_any_regex(path, ["/.rar$/g"]): $ unrar x "{path}" "{target}" $?
            contains_any_regex(path, ["/.rpm$/g"]): $ rpm2cpio "{path}" | cpio -idm $?
            contains_any_regex(path, ["/.tar$/g"]): $ tar xf "{path}" -C "{target}" $?
            contains_any_regex(path, ["/.xz$/g"]): $ xz --decompress "{path}" $?
            contains_any_regex(path, ["/.7z$/g"]): $ 7z -y "{path}" -o "{target}" $?
            contains_any_regex(path, ["/.zip$/g", "/.war$/g", "/.jar$/g"]): $ unzip "{path}" -d "{target}" $?
```

Reply:
this check return an error but I think that is a bug of Amber as in that case we are not using a function but executing a command.
can you check @Ph0enixKM?